oak_restricted_kernel/src/mm/page_tables.rs

```diff
@@ -267,3 +267,35 @@ impl Translator for RootPageTable {
         self.inner.translate_physical_frame(frame)
     }
 }
+
+/// Wrapper struct that holds the current page table is there one.
+pub struct CurrentPageTable {
+    inner: Option<RootPageTable>,
+}
+
+impl CurrentPageTable {
```

Comment:
This doesn't seem to add much over just using `Option<RootPageTable`, apart from changing the names of some functions. Do you plan on adding more functionality to this wrapper in future?

Reply:
I would expect it to later on update the pagetable on the CPU level upon updating. 

The other difference is that it cannot be set back to `None`.